napi/scripts/napi.js

```diff
@@ -24,12 +24,29 @@ const moduleName = path.basename(process.cwd());
 process.env.NODE_INCLUDE_PATH = nodeIncludePath
 process.env.NODE_MAJOR_VERSION = nodeMajorVersion
 
+const platform = os.platform()
+var libExt = '.so'
+
+switch(platform) {
+	case 'darwin':
+		libExt = '.dylib'
+		break;
+	case 'linux':
```

Comment:
Maybe add `linux` as last case and let it fall-through because value is same for linux and default?